### PR TITLE
Fixes #1189 Nhibernate update

### LINKED ISSUE #1189: Until we update NHibernate, we should add log4net.dll in distribution folder
or we update NHibernate. I can try and see if our tests are still passing ??

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

**Comment by msevestre** [comment]
@Yuri05 I just want to run all the test using this branch to see if anything breaks. I did not have to change any code except one method that was removed in the test

**Comment by msevestre** [comment]
IF this is successful, we can decide to merge or not. This should solve the issue with dependency on log4net that is not even used

**Comment by msevestre** [comment]
@Yuri05 I ran all of the tests locally as well (nightly tests)
Looks good
image

File: src/PKSim.Matlab/ApplicationStartup.cs [hunk 1]
```diff
@@ -47,29 +47,27 @@ private void initializeForMatlab()
 
       private static void redirectNHibernateAssembly()
       {
-         redirectAssembly("NHibernate", new Version(4, 1, 0, 4000), "aa95f207798dfdb4");
+         redirectAssembly("NHibernate", new Version(5, 2, 0, 0), "aa95f207798dfdb4");
```
Comment (Yuri05) [inline src/PKSim.Matlab/ApplicationStartup.cs]: what is this aa95f207798dfdb4 ?

Comment (msevestre) [reply src/PKSim.Matlab/ApplicationStartup.cs]: This is the nhiberbate package id (see app.config). This code ensures that the right dll is loaded. We had issues when loading from Matlab

**Comment by Yuri05** [comment]
@msevestre I cannot say if there is something wrong or not. I didn't see anything suspicious, so if you think that me can merge it - do it.

**Comment by msevestre** [comment]
For me it also looks legit. All tests are passing so I think we should do it

**Pull request merged**